simapp/app_test.go

```diff
@@ -45,3 +48,82 @@ func TestGetMaccPerms(t *testing.T) {
 	dup := GetMaccPerms()
 	require.Equal(t, maccPerms, dup, "duplicated module account permissions differed from actual module account permissions")
 }
+
+func TestRunMigrations(t *testing.T) {
+	db := dbm.NewMemDB()
+	app := NewSimApp(log.NewTMLogger(log.NewSyncWriter(os.Stdout)), db, nil, true, map[int64]bool{}, DefaultNodeHome, 0, MakeTestEncodingConfig(), EmptyAppOptions{})
+
+	// Create a new configurator for the purpose of this test.
+	app.configurator = module.NewConfigurator(app.MsgServiceRouter(), app.GRPCQueryRouter(), app.keys)
+
+	testCases := []struct {
+		name         string
+		moduleName   string
+		expRegErr    bool // errors while registering migration
+		expRegErrMsg string
+		expRunErr    bool // errors while running migration
+		expRunErrMsg string
```

Comment:
The `Makefile` does not yet support `-short`. Plus, this test case does not seem to be too heavy

Reply:
> Tip: to reduce the amount of parameters here, we can remove expRegErr and expRunErrMsg, and use expRegErrMsg != "" and expRunErrMsg != ""

I actually don't like `errMsg != ""`, because you don't know which error path you're testing, and it happened to me that something errored but for a different reason.